### PR TITLE
T8509 - add breadcrumbs

### PR DESCRIPTION
**Pull request by Benas-Burneika**

### CODE REVIEW

**Comment by bmarty92** [comment]
did You check `gatsby-plugin-breadcrumb` ?

File: src/components/Breadcrumbs/Breadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    }
+  };
+
+  return (
+    <nav className="Breadcrumb">
+      <ul className="Breadcrumb__container">
+        {crumbs.map((crumb, index) => {
+          return (
+            <li key={index} className="Breadcrumb__container--crumb">
+              <button
```
Comment (grinkus-adapt) [inline src/components/Breadcrumbs/Breadcrumb.js:20]: Couldn't these be simple `Link`s?

**Review by grinkus-adapt — CHANGES_REQUESTED** [review]

File: src/components/Breadcrumbs/Breadcrumb.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            </li>
+          );
+        })}
+      </ul>
+    </nav>
+  );
+};
+
+Breadcrumb.propTypes = {
+  crumbs: PropTypes.array,
```
Comment (bmarty92) [inline src/components/Breadcrumbs/Breadcrumb.js:36]: expand this prop type 
example: `PropTypes.arrayOf(PropTypes.number)`

File: src/pages/bukime-budrus/patikima-informacija.js [hunk 3]
```diff
@@ -29,6 +30,7 @@ const Page = ({ data }) => {
 
       {!!content && (
         <Constraint>
+          <Breadcrumb crumbs={[`Titulinis`, `Patikima informacija`]} />
```
Comment (bmarty92) [inline src/pages/bukime-budrus/patikima-informacija.js:33]: maybe it's better to move crumbs properties to separate const example `crumbsData` instead passing whole array

**Pull request merged**